### ISSUE TITLE
Remove YAML mapping examples

### ISSUE THREAD

**Issue by xabbuh** [symfony/symfony-docs#6752]
Given that Doctrine will remove support for defining ORM metadata in YAML format (see doctrine/doctrine2#5932), we should probably already remove references to this format.

**Comment by ogizanagi** [comment]
@xabbuh : Sorry for the non-related comment, but is there any related issue or comment explaining why Doctrine will not support anymore defining ORM metadata in YAML formats ?

**Comment by xabbuh** [comment]
@ogizanagi Somewhere I read a comment by @Ocramius that the code for it was too fragile IIRC. Maybe he can give you some more insights.

**Comment by Ocramius** [comment]
The change will affect ORM 3.0: do you really wabt to raise it here already? I don't think that you should worry about it yet.

**Comment by xabbuh** [comment]
@Ocramius Will the XML and annotation mapping formats change in 3.0? If not, it imo is a good idea not to advocate to use YAML anymore to ease migrating to Doctrine 3.

**Comment by Ocramius** [comment]
Not sure yet.

**Comment by robfrawley** [comment]
@Ocramius Where can I find discussions on the removal of YML in D3?!

**Comment by Ocramius** [comment]
@robfrawley we are currently working on rewriting the entire metadata-related internals of the ORM, and the Yaml driver was just long-due removal. We just needed to finally have a practical excuse (3.x + refactoring) to do so, and we did.

The Yaml driver has always been lagging back (bugs, support, docs, etc) due to the squishy format of it (no type checking, no static analysis, no autocompletion), so you'll likely see it implemented by third parties than by Doctrine itself in 3.x.
